packages/react/scripts/generateIcons.js

```diff
@@ -6,26 +6,68 @@ const dirPath = `./src/primitives/Icon/icons/`;
 const iconSetPath = '../ui/src/icons/*.svg';
 const iconNames = [];
 
-const template = ({ iconName, source }) => `import classNames from 'classnames';
-
-import { ComponentClassNames } from '../../shared';
-import { View } from '../../View';
-
-export const ${iconName} = (props) => {
-  const { className, ...rest } = props;
-  return (
-    <View
-      as='span'
-      width='1em'
-      height='1em'
-      className={classNames(ComponentClassNames.Icon, className)}
-      {...rest}
-    >
-      ${source.replace('black', 'currentColor')}
-    </View>
-  );
-}
+const INTERNAL_ICONS = [
+  'Icon',
```

Comment:
This list comes from the icons that will remain after the 3.0 release:
https://github.com/aws-amplify/amplify-ui/blob/3.0/packages/react/__tests__/exports.ts#L36-L55